ext/LegendGeSimExt.jl

```diff
@@ -0,0 +1,567 @@
+module LegendGeSimExt
+using LegendGeSim
+using PropDicts
+using LegendGeSim
+using LegendGeSim: Environment
+using MJDSigGen
+using MJDSigGen: SigGenSetup
+using LegendGeSim: SiggenSimulator
+using ArgCheck
+using ArraysOfArrays
+using CurveFit
+using DelimitedFiles
+using Distributions
+using DSP 
+using ElasticArrays
+using HDF5
+using IntervalSets
+using JSON
+using LegendDataManagement
+using LegendDataTypes
+using LegendHDF5IO
+using LegendTextIO
+using LinearAlgebra
+using Parameters
+using Polynomials
+using PropDicts
+using RadiationDetectorDSP 
+using RadiationDetectorSignals
+using RadiationSpectra
+using Random
+using Random123
+using RecipesBase
+using Requires
+using SignalAnalysis
+using SolidStateDetectors
+using StaticArrays
+using Statistics
+using StatsBase
+using StructArrays
+using Tables
+using TypedTables
+using Unitful
+
+import LsqFit
+
+using SolidStateDetectors: ConstructiveSolidGeometry as CSG
+
+
+####################################
+### FieldGen
+####################################
+"""
+    SiggeSimulator(sim_conf::PropDict)
+
+    LegendGeSimConfig -> SiggenSimulator
+
+Construct SiggeSimulator instance based on simulation
+    configuration given in <sim_conf>.
+"""
+
+function SiggenSimulator(simulation_settings::PropDict)
```

Comment:
Does this need to be `LegendGeSim.SiggenSimulator`?

Reply:
I don't have any siggen code in the src code of LegendGeSim. I have tried to import all the siggen code in the ext folder. Is that okay?